app/src/main/java/com/example/triptracker/view/map/MapPopups.kt

```diff
@@ -469,7 +491,7 @@ fun PinDescription(descriptionsOpen: MutableState<List<Boolean>>, pin: Pin, inde
               .fillMaxWidth()) {
         val displayText =
             if (!descriptionsOpen.value[index]) {
-              pin.name.take(30) + if (pin.name.length > 30) "..." else ""
+              pin.name.take(20) + if (pin.name.length > 20) "..." else ""
```

Comment:
could you put this value in a global variable like `maxPinNameLen`. thanks

Reply:
Done !